curriculum/challenges/espanol/02-javascript-algorithms-and-data-structures/functional-programming/combine-two-arrays-using-the-concat-method.md

```diff
@@ -1,45 +1,46 @@
 ---
 id: 587d7da9367417b2b2512b66
-title: Combine Two Arrays Using the concat Method
+title: Combina dos arreglos utilizando el método "concat"
```

Comment:
Should `concat` be in quotes here?

Reply:
The Spanish translation team have been using double quotes around key words for emphasis.  I am not sure it is necessary, but that is the current approach.  This is just one of many titles that are written like this.

Reply:
I'm not sure how I feel about that approach but that can be discussed with the team in chat or something. I'll approve the PR to avoid blocking.